functions.sh

```diff
@@ -100,8 +100,11 @@ validate_RESULT() {
     # logs from Anaconda - whole /var/log/anaconda/ directory is copied out,
     #                      this can be used for saving specific test output
     # anaconda.coverage
-    # RESULT file from the test
+    # anaconda-ks.cfg - the kickstart saved after installation, useful for
+    #                   debugging
+    # RESULT - file from the test
     for item in /root/anaconda.coverage \
```

Comment:
Could you remove `anaconda.coverage`, please? The file no longer exists.

Reply:
Done.